ios/FluentUI/Command Bar/CommandBar.swift

```diff
@@ -39,16 +37,30 @@ open class CommandBar: UIView, FluentUIWindowProvider {
             self.trailingButton = button(forItem: trailingItem, isPersistSelection: false)
         }
 
-        backgroundColor = commandBarTokens.backgroundColor
+        backgroundColor = UIColor(colorSet: tokens.backgroundColor)
         translatesAutoresizingMaskIntoConstraints = false
 
+        self.themeObserver = NotificationCenter.default.addObserver(forName: Notification.Name.didChangeTheme,
+                                                               object: nil,
+                                                               queue: .main) { [weak self] _ in
+            if let strongSelf = self {
+                strongSelf.updateButtonTokens()
```

Comment:
so notifcation can be posted to all windows, but updatebuttontokens will figure/filter out if the theme changes for that window is applicable for this instance of the class?

Reply:
Good point. At the moment no, this is pretty brute-force logic, and will apply to the entire app. But that's a good optimization: the theme posting could/should include the window, and then we can have logic that decides whether it applies to us or not.